lib/web_ui/test/engine/path_metrics_test.dart

```diff
@@ -4,11 +4,11 @@
 
 import 'dart:math' as math;
 
-import 'package:test/bootstrap/browser.dart';
-import 'package:test/test.dart';
-import 'package:ui/ui.dart';
+import 'package:test/bootstrap/browser.dart'; // ignore: import_of_legacy_library_into_null_safe
```

Comment:
Question: Do we need to change every occurrence of this import? I think we use it many golden tests, example: https://github.com/flutter/engine/blob/master/lib/web_ui/test/golden_tests/engine/canvas_blend_golden_test.dart

Reply:
Only if the library that imports it is Null Safe, i.e. the Dart SDK is `2.12.0`, and the library does not opt-out from null safety. The example you pointed at does opt-out, it has `// @dart = 2.6`, so sets the language version to `2.6`, which is less than `2.12` where `non-nullable` feature it released, and less than `2.10` when this feature was experimentally-released.